index.html

```diff
@@ -48,18 +47,18 @@ <h2>Life Story</h2>
       </section>
     </main>
 
-    <!--
+    <!---
 
-  Hello Camper!
+Hello Camper!
 
-  For now, the test suite only works in Chrome!
-  Please read the README below in the JS Editor before beginning.
-  Feel free to delete this message once you have read it.
-  Good luck and Happy Coding!
+For now, the test suite only works in Chrome!
+Please read the README below in the JS Editor the before beginning.
+Feel free to delete this message once you have read it.
+Good luck and Happy Coding!
 
-  - The freeCodeCamp Team
+- The freeCodeCamp Team
 
-  -->
+--->
```

Comment:
- It is good practice not to leave any comments in your code base. Don't use comments as an excuse for unclean code This might be OK as a temporary fix, but don't make it standard practice. Instead, strive to make your code itself as clean and readable as possible. ♻️

Reply:
- [ ] Let's be more polite, what do you think? Because your job here is to help another person improve in a friendly way. So let's use `I think it would be better to not...` instead of every `don't`s in your sentences.